docker/cami-medical-compliance/docker-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Using hardcoded values for paths for now.
+# TODO(@iprunache): find a way to make the paths configurable
+
+# Doing a migration at the container start is not a good idea on the long term
+# as usually one wants containers to start as fast as possible.
+python /cami-project/medical_compliance/manage.py migrate
```

Comment:
@iprunache that's one reason, and another reason is the principle of least surprise - you wouldn't expect a server starting up to run migrations. In production, you usually want to control very tightly when migrations are run, and it's usually in maintenance mode for the whole product if there's a major schema change.